README.md

```diff
@@ -1,25 +1,32 @@
+# Domain Model for a School
 
+## Objectives
 
-# Domain Model for a School
+1. Define the concept of a domain model. 
+2. Create your own domain model. 
+3. Write instance methods that manipulate nested data structures. 
+
+## What is a Domain Model?
 
-In this assignment you'll be writing a simple app for a school. 
+A domain model is a representation of real-world concepts in software. The concept of domain modeling is key in object orientation. In object orientation, we think of our classes as templates for objects. The instances of our classes are thought of as objects. For example, a Person class produces people objects that have attributes and behaviors, as described and enacted by instance methods. 
```

Comment:
Technically a `Person` class generates `Person` objects. It's a bit non english-y but that's ok